manageability-docker/TBH/inbm-vision-third-party-programs.txt

```diff
@@ -0,0 +1,610 @@
+Bit Creek Third Party Program Files
```

Comment:
Can remove this one and just use the third-party-programs.txt file only.  We are only using one file now for the product as a whole instead of 2 separate since the Open source.  https://github.com/intel/intel-inb-manageability/blob/develop/third-party-programs.txt